src/components/screens/LunchSection.tsx

```diff
@@ -76,10 +76,16 @@ const parseAttributes = (input?: string) => {
 const LunchOffer = ({ name, price }: { name: string; price: string }) => (
   <View style={styles.container}>
     <View style={styles.nameContainer}>
-      <RegularText accessibilityLabel={`Mittagstisch (Gerichtname): ${name}`}>{name}</RegularText>
+      <RegularText
+        accessibilityLabel={`${texts.eventRecord.description} ${consts.a11yLabel.lunch}`}
```

Comment:
this seems o not resolve to `Mittagstisch (Gerichtname): ${name}`? please have a check

Reply:
done in d906981aa68e8a03252497b159ecff3df02be64f